src/meshdb/templates/rest_framework/login_base.html

```diff
@@ -0,0 +1,12 @@
+{% extends "rest_framework/login_base.html" %}
+
+{% block branding %}
+   <h3 style="margin: 0 0 5px;">MeshDB Data API Login</h3>
+    <p  style="margin: 0 0 20px; font-size: 12px">
+    <i>
+        Admin UI credentials work here, but non "staff" users who don't have admin
```

Comment:
Wait, do we need to give people an account so they can access our docs? I'd rather they be totally open. If that's a security concern, we should address the security concern so our docs can be open. Make it more accessible that way.

Reply:
Credentials are only necessary to use the interactive "call this endpoint now" features of `swagger-ui`. They are not necessary to view the endpoint & schema descriptions, or to query any public endpoints